Database/Updates/World/01_05_21_2017_viewUpdate.sql

```diff
@@ -0,0 +1,227 @@
+/*
+SQLyog Ultimate v12.4.1 (64 bit)
+MySQL - 10.1.23-MariaDB : Database - ace_world
+*********************************************************************
+*/
+
+/*!40101 SET NAMES utf8 */;
+
+/*!40101 SET SQL_MODE=''*/;
+
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+CREATE DATABASE /*!32312 IF NOT EXISTS*/`ace_world` /*!40100 DEFAULT CHARACTER SET utf8 */;
+
+USE `ace_world`;
+
+/*Table structure for table `vw_ace_object` */
+
+DROP TABLE IF EXISTS `vw_ace_object`;
+
+/*!50001 DROP VIEW IF EXISTS `vw_ace_object` */;
```

Comment:
I think you may have to hand edit the scripts, as these pseudo-comments are acceptable syntax, as far as MySQL and MariaDB are concerned.